src/components/AppNavigation/EditCalendarModal.vue

```diff
@@ -24,7 +24,19 @@
 					:placeholder="$t('calendar', 'Calendar name …')"
 					@input="calendarNameChanged = true">
 			</div>
-
+			<template v-if="canBeShared">
+				<NcListItem :name="$t('calendar', 'All events in this calendar show you as busy by default')"
+					@click="isTransparent = !isTransparent">
+					<template #icon>
```

Comment:
[NcCheckboxRadioSwitch](https://nextcloud-vue-components.netlify.app/#/Components/NcCheckboxRadioSwitch) can be used here instead